librz/include/rz_search.h

```diff
@@ -213,13 +231,31 @@ typedef enum {
 	RZ_SEARCH_COLLECTION_CRYPTOGRAPHIC_ECC,
 	RZ_SEARCH_COLLECTION_CRYPTOGRAPHIC_SAFECURVES,
 	RZ_SEARCH_COLLECTION_CRYPTOGRAPHIC_X509,
+	RZ_SEARCH_COLLECTION_CRYPTOGRAPHIC_END_CRYPTO,
+	RZ_SEARCH_COLLECTION_CRYPTOGRAPHIC_ALL_KEY_BASED, ///< All key based algos from above.
+
+	RZ_SEARCH_COLLECTION_CRYPTOGRAPHIC_BEGIN_HASH, ///< Hash related algorithms
+	RZ_SEARCH_COLLECTION_CRYPTOGRAPHIC_ENTROPY, ///< Block entropies.
+	RZ_SEARCH_COLLECTION_CRYPTOGRAPHIC_HASH, ///< Block hashes.
+	RZ_SEARCH_COLLECTION_CRYPTOGRAPHIC_END_HASH,
+
```

Comment:
TBH, i would have a different collection for hash and entropy.

Reply:
Never mind, I remove the find() data from the crypto search. There is no need to have dead code for "maybe usage".